portal/templates/portal/base.html

```diff
@@ -168,13 +168,16 @@
 </div>
 {% endblock topBar %}
     {% block subNav %}
-    {% endblock subNav %}
     <div id="messages">
         {% if messages %}
         <div class="sub-nav--message">
             {% for message in messages %}
             <div class="sub-nav {{ message.tags }}">
+                {% if 'warning' in message.tags %}
+                <img title="Warning" alt="Warning sign" src="{% static 'portal/img/warning.png' %}">
```

Comment:
does this get used if the message tag is error?

Reply:
+1